README.md

```diff
@@ -1,34 +1,48 @@
 # Bank Admin
 
-SSCL staff facing site for Prisoner Money suite of apps.
+SSCL staff facing site for [Prisoner Money suite of apps](https://github.com/ministryofjustice/money-to-prisoners).
+
+## Requirements
+
+- Unix-like platform with Python 3.6+ and NodeJS 10 (e.g. via [nvm](https://github.com/nvm-sh/nvm#nvmrc))
 
 ## Running locally
 
 It's recommended that you use a python virtual environment to isolate each application.
 Please call this `venv` and make sure it's in the root folder of this application so that
```

Comment:
Just adding a note the comments on https://github.com/ministryofjustice/money-to-prisoners-api/pull/600 probably apply here too (only if anything changes of course).